dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/category/DefaultCategoryService.java

```diff
@@ -310,6 +304,11 @@ public Set<CategoryOption> getCoDimensionConstraints(User user) {
     return options;
   }
 
+  @Override
+  public List<CategoryOption> getCategoryOptionsByUid(List<String> catOptionUids) {
+    return idObjectManager.getByUid(CategoryOption.class, catOptionUids);
```

Comment:
I think there is a "typed" helper in `categoryStore` that does the same which we should prefer to use

Reply:
There isn't one for `CategoryOption`. I will add one to `CategoryOptionStore` though.